### DISCUSSION TITLE
using "GUI_TAB_PLAYER"

### DISCUSSION THREAD

**Discussion by Deadlineem** [YimMenu/YimMenu#2748] [Q&A]
when i use gui.get_tab("GUI_TAB_PLAYER") it does nothing.  Is there a special way to use this thats not in the documentation?

Trying to put more options under the selected player

@Yimura

**Comment by Deadlineem** [comment]
> Not posting your entire script, tagging Yim for no reason at all (he doesn't even lead the Lua team) and just asking ridiculous questions, on top of your recent behavior, you're becoming a determent to us, and a massive nuisance.

1. Im just trying to put things under the tab, im not posting it because its toxic options and I assumed that wasnt allowed.
2. I didnt know he didnt lead the lua team, im new to this as i have said multiple times.
3. My recent behavior?  I've only posted issues and asked questions, the only two people i know for anything in this menu are yimura and yourself, im sorry you disagree with the things i have posted or asked about.  

Forget i bothered to ask frfr

and btw, where exactly am I supposed to ask questions that im genuinely needing help with if not here?  I post it in issues, you complain about it, so i went to discussions to get help without creating useless issues under the issues tab and you complain about it.

**Reply by Yimura** [reply]
There are just multiple people working on the team and @'ing me will not change how quickly I see it or make it more or less likely to answer. I'll see it either way and answer whenever.

**Reply by Deadlineem** [reply]
Well, i apologize, but this may also be a bug too because if i use GUI_TAB_PLAYER, it does nothing but when i do gui.get_tab("") it puts it under the selected player, but also creates a box for it when the menu is loaded/opened it just pops up.

Hence why i asked in the first place.  I cannot contribute my findings or help or even learn a thing without your help or the help of someone who is directly involved in the project.  Im not a VIP, you arent required to answer me immediately, I only tag you for reference and if it was an issue to begin with, you could have just told me in the first place and I wouldnt have tagged anybody.

![Screenshot_326](https://github.com/YimMenu/YimMenu/assets/24372625/0588664c-f84a-406b-a0b4-4df78dfbaf2d)

**Comment by xiaoxiao921** [comment]
The empty string tab is an oddity that isn't documented, but yeah it lands you to the player tab. A list of possible tab is available in the repo docs folder https://github.com/YimMenu/YimMenu/blob/master/docs/lua/tabs.md

**Reply by Yimura** [reply]
Maybe we should just add an extra function that grabs the player tab and does that hidden behaviour of using the tab with `""` as name.

**Reply by Deadlineem** [reply]
I know.  I was using that to find the tabs, it seems like the missions tab and other tabs work but the GUI_TAB_PLAYER portion does not work, and using nothing points to what i assume is supposed to be GUI_TAB_PLAYER

**this is what it shows along with the other screenshot above when i use gui.get_tab("")**

it generates both the top screenshot when you load and open the menu (only when you dont click a tab, when you click a tab it doesnt show again, oddly) and then it generates this screenshot below as well when you click a player.

Nothing happens at all when i use GUI_TAB_PLAYER instead of leaving it blank.
![Screenshot_327](https://github.com/YimMenu/YimMenu/assets/24372625/8f138ecb-3d28-4e3a-9ed2-944fac77139f)

**Comment by Deadlineem** [comment]
Has anyone ever figured out a way to fix this tab issue?  
![Screenshot_377](https://github.com/YimMenu/YimMenu/assets/24372625/9e6f4731-8c28-4af5-b48b-b9632b4cb3d7)

**Comment by malloc2k** [comment]
Yo @Deadlineem ur script seems awsome, I was wondering: 

Any bans from anyone that use your script? :) 
Thanks

**Reply by Deadlineem** [reply]
I have quite a bit of people using the addon in my discord channel, tossing ideas around and such.  There has only been 2 people banned that I am aware of, one of them was my other contributor - USBMenus - he was messing with trigger_script_events while dev testing and landed himself a ban

The other guy who just got banned caught a 30 day ban, he said the email said "misuse of game functions"

-- My advice:  Its all about moderation, I have been playing/coding/testing for a solid 3 months with no issues and using YimMenu for around 3 years with no bans.  Even though I do have toxic options in the addon, I do not really use them often.

Also, since extras addon is not directly a part of YimMenu itself, post on my github discussions instead of here, to avoid issues with the creators of YimMenu.

**Reply by malloc2k** [reply]
Aight, thanks my man

**Comment by Deadlineem** [comment]
@Yimura Adding GUI_TAB_PLAYER adds an entirely new tab to the menu with that name when you use it.
